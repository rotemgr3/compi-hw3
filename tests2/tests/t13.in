//piazza 80 - should be errorDef +
void main() {int a = 5;}
int main (int a) {int booga = 5;}