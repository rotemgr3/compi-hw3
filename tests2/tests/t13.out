---end scope---
a INT 0
line 3: identifier main is already defined
